tests/main/media-sharing/task.yaml

```diff
@@ -4,17 +4,16 @@ details: |
     the mount namespace used by snap applications into the main mount
     namespace.
 prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local_devmode test-snapd-tools
     mkdir -p /media/src
     mkdir -p /media/dst
     touch /media/src/canary
-    snap install --devmode snapd-hacker-toolbelt
 execute: |
-    export PATH=$PATH:/snap/bin
     test ! -e /media/dst/canary
-    snapd-hacker-toolbelt.busybox sh -c 'mount --bind /media/src /media/dst'
+    test-snapd-tools.cmd mount --bind /media/src /media/dst
```

Comment:
I see that tests/lib/snaps/test-snapd-tools exists, it has 'cmd' and you are using install_local. The disparity between this and test-snapd-tools in the store is confusing. Can the one in the store be removed?